src/App.js

```diff
@@ -1,10 +1,15 @@
 import "./App.css";
+import { Routes, Route } from "react-router-dom";
 import { Homepage } from "./pages/Homepage/Homepage";
+import { Wishlist } from "./pages/Wishlist/Wishlist";
```

Comment:
Keep an `index.js` file in pages and export every page from there.
So when you'll actually import the pages they'll get imported as - 
`import {Homepage, Wishlist, Cart, Products, Login, SIgnup } from "./pages"`

Reply:
> Keep an `index.js` file in pages and export every page from there. So when you'll actually import the pages they'll get imported as - `import {Homepage, Wishlist, Cart, Products, Login, Signup } from "./pages"`

Great suggestion @sudipkundu999 ! Will do this 👍🏻